packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -43,6 +43,8 @@ export interface TabsProps extends Omit<React.HTMLProps<HTMLElement | HTMLDivEle
   isVertical?: boolean;
   /** Enables no border bottom tab styling */
   hasBorderBottom?: boolean;
```

Comment:
```suggestion
  hasNoBorderBottom?: boolean;
```

Based on the prop description, I'm guessing you tested a `hasNoBorderBottom` prop, which is what we want based on the core docs: https://patternfly-pr-4774.surge.sh/components/tabs/ 

The tabs have a bottom border by default, so it makes sense to have a prop that lets a consumer remove the bottom border.

Reply:
Yeah `hasBorderBottom` was originally `hasNoBorderBottom` in #7183 when I first PR'd it. @kmcfaul @nicolethoen I'd appreciate your thoughts on this suggestion. I'm a fan but since you both advocated for `hasBorderBottom` I'm hesitant to change it before getting your input.

Thank you for putting this much thought into it!

Reply:
Ah okay, thank you for linking the comment thread @wise-king-sullyman and for sharing your input @kmcfaul !  

Avoiding the double negative makes sense. If that's the case then we would want to keep both `hasBorderBottom` and `hasSecondaryBorderBottom` for the styling to work as expected. I've added a suggestion to update the prop description which would clarify the use case of `hasBorderBottom` a bit better - WDYT @wise-king-sullyman?

Reply:
That suggestion looks great to me @jenny-s51, thank you for doing it!